test/controllers/users/sessions_controller_test.rb

```diff
@@ -124,4 +124,59 @@ class Users::SessionsControllerTest < ActionDispatch::IntegrationTest
     assert_template :new
     assert_match "Invalid Email or password.", flash[:alert]
   end
+
+  test 'redirects to private cms-page after sign-in successfully' do
+    Apartment::Tenant.switch @restarone_subdomain.name do
+      @restarone_user.update!(can_access_admin: false)
```

Comment:
@alis-khadka double check this test case, feels like a false positive. The user doesn't have permission to access private pages